modules/misc/news.nix

```diff
@@ -779,6 +779,12 @@ in
           A new module is available: 'programs.offlineimap'.
         '';
       }
+
+      { time = "2018-09-13T16:21:00+00:00";
```

Comment:
Please match the formatting used elsewhere in the file.

Reply:
I've moved the time onto its own line.  Should I also update the time stamp?  I'm not sure exactly when this should count as having been added.

Reply:
No worries, I can do that.